internal/converter/internal/staticconvert/testdata/traces.yaml

```diff
@@ -10,157 +10,27 @@ traces:
-                  type: boolean_attribute,
-                  boolean_attribute: {key: key4, value: true}
-            },
-            {
-                  name: test-policy-11,
-                  type: ottl_condition,
-                  ottl_condition: {
-                      error_mode: ignore,
-                      span: [
-                            "attributes[\"test_attr_key_1\"] == \"test_attr_val_1\"",
-                            "attributes[\"test_attr_key_2\"] != \"test_attr_val_1\"",
-                      ],
-                      spanevent: [
-                            "name != \"test_span_event_name\"",
-                            "attributes[\"test_event_attr_key_2\"] != \"test_event_attr_val_1\"",
-                      ]
-                  }
-            },
-            {
-                name: and-policy-1,
-                type: and,
-                and: {
-                  and_sub_policy: 
-                  [
-                    {
-                      name: test-and-policy-1,
-                      type: numeric_attribute,
-                      numeric_attribute: { key: key1, min_value: 50, max_value: 100 }
-                    },
-                    {
-                        name: test-and-policy-2,
-                        type: string_attribute,
-                        string_attribute: { key: key2, values: [ value1, value2 ] }
-                    },
-                  ]
-                }
-            },
-            {
-                name: composite-policy-1,
-                type: composite,
-                composite:
-                  {
-                    max_total_spans_per_second: 1000,
-                    policy_order: [test-composite-policy-1, test-composite-policy-2, test-composite-policy-3],
-                    composite_sub_policy:
-                      [
-                        {
-                          name: test-composite-policy-1,
-                          type: numeric_attribute,
-                          numeric_attribute: {key: key1, min_value: 50, max_value: 100}
-                        },
-                        {
-                          name: test-composite-policy-2,
-                          type: string_attribute,
-                          string_attribute: {key: key2, values: [value1, value2]}
-                        },
-                        {
-                          name: test-composite-policy-3,
-                          type: always_sample
-                        }
-                      ],
-                    rate_allocation:
-                      [
-                        {
-                          policy: test-composite-policy-1,
-                          percent: 50
-                        },
-                        {
-                          policy: test-composite-policy-2,
-                          percent: 25
-                        }
-                      ]
-                  }
-              },
-            ]
+          ]
```

Comment:
let's make this test a little less insane to understand by removing a bunch of config already tested inside the otel converter.